frontend/src/pages/projects/notebook/NotebookStatusToggle.tsx

```diff
@@ -71,22 +70,10 @@ const NotebookStatusToggle: React.FC<NotebookStatusToggleProps> = ({ notebookSta
         </FlexItem>
         <FlexItem>
           <NotebookStatusPopover
```

Comment:
Thought we agreed to rename this to `NotebookStatusText`?

Reply:
Ah, I thought you were talking about the name of the scss file lol... will rename it.

Reply:
SCSS files ALWAYS match component filed names -- unless they are global things.